packages/wallet/frontend/src/pages/account/create.tsx

```diff
@@ -135,10 +135,12 @@ export const getServerSideProps: GetServerSideProps<{
     }
   }
 
-  const assets = response.data?.map((asset) => ({
-    value: asset.id,
-    label: asset.code
-  }))
+  const assets = response.data
+    ?.filter((asset) => asset.scale === 2)
+    ?.map((asset) => ({
+      value: asset.id,
+      label: asset.code
+    }))
```

Comment:
To only display Rapyd supported assets and exclude the high scale ones.

Reply:
should it be less or equal with 2? (as in some cases in can be 0)